geode-core/src/test/java/org/apache/geode/internal/cache/FireAndForgetFunctionOnAllServersDUnitTest.java

```diff
@@ -92,22 +90,23 @@ public void testFireAndForgetFunctionOnAllServers() {
 
       // Step 4. Execute the function to put DistributedMemberID into above created replicated
       // region.
-      Function function =
-          new TestFunction(false, TestFunction.TEST_FUNCTION_FIREANDFORGET_ONALL_SERVERS);
+      Function function = new FireAndForgetFunctionOnAllServers();
       FunctionService.registerFunction(function);
 
+      PoolImpl pool = (PoolImpl) pool1;
+
+      await().atMost(60, SECONDS)
+          .until(() -> Assert.assertEquals(1, pool.getCurrentServers().size()));
```

Comment:
The 2 lines should be moved to line 120. Otherwise, I wonder why the test will pass.